.docforge/manifest.yaml

```diff
@@ -5,5 +5,60 @@ structure:
   properties:
     frontmatter:
       title: Machine Controller Manager
+      weight: 1
       description: Declarative way of managing machines for Kubernetes cluster
-  source: /README.md
\ No newline at end of file
+  source: /README.md
+- name: development
+  properties:
+    frontmatter:
+      title: Development
+      weight: 2
+      categories:
+        - Developers
+  nodesSelector:
+    path: /docs/development
+- name: documents
+  properties:
+    frontmatter:
+      title: Documents
+      weight: 3
+      categories:
+        - Developers
```

Comment:
Just a question about how the categories are useful ? Are they used by `docforge` internally somehow, because I don't see any effect of these categories (which were also available in `_index.md`) in the `gardener.cloud` website

Reply:
@himanshu-kun  See initial invitation/info e-mail with screenshot. Yes, the categories shall be made available on the documentation pages:
- Externally: Visitors can switch the persona to see different docs, for developers, operators, and/or users
- Internally: Only the user persona tagged docs will be shown, because developers and operators are not in scope of that page